# implements SBP message MSG_GNSS_TIME_OFFSET

int16 weeks         # Weeks portion of the time offset
int32 milliseconds  # Milliseconds portion of the time offset
int16 microseconds  # Microseconds portion of the time offset
uint8 flags         # Status ï¬‚ags (reserved)
